OasisDEX/oasis-borrow
# [sc-7467] [UI] Add marketing details to landingpage for Oasis Multiply using Aave

# [[UI] Add marketing details to landingpage for Oasis Multiply using Aave](https://app.shortcut.com/oazo-apps/story/7467/ui-add-marketing-details-to-landingpage-for-oasis-multiply-using-aave)


## Changes 👷‍♀️
- changed the home banner box
- added new tag "Automation Enabled" for wbtc-a, -b, -c, and eth-a,-b,-c, YFI-A, LINK-A
- added copy for the new Aave Multiply feature
  
## How to test 🧪
- check homepage changes (banner box, product card tags, aave multiply info on multiply tab)
- check /multiply changes (product card tags, aave multiply info)

## Todo:
- KB links
